console/v1/types_console_notification.go

```diff
@@ -0,0 +1,50 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Extension for configuring openshift web console notifications.
+type ConsoleNotification struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              ConsoleNotificationSpec `json:"spec"`
+}
+
+type ConsoleNotificationSpec struct {
+	// Text of the notification
+	Text string `json:"text"`
+	// The location of the notification (top, bottom, topBottom)
```

Comment:
Is this optional? I think we can default to top.

Reply:
The possible values no longer match

Reply:
I'll cut that bit of the comment so we don't have to maintain it.  The type is below & easy enough to look up.

Reply:
& annotate.

Reply:
It might be good to keep so it shows up the in the YAML editor sidebar.